bridge/bridge.go

```diff
@@ -68,6 +68,10 @@ func (b *Bridge) joinChannels(channels map[string]config.ChannelInfo, exists map
 	return nil
 }
 
+func (b *Bridge) GetConfigFile() string {
```

Comment:
Not needed

Reply:
As in, just load the viper global in bridge/bridge.go?